KomaMRIBase/src/datatypes/sequence/Grad.jl

```diff
@@ -182,10 +182,12 @@ getproperty(x::Matrix{Grad}, f::Symbol) = begin
 	end
 end
 
-# Gradient comparison
+"""
+For comparing two `Grad`s custom types
+"""
 Base.isapprox(gr1::Grad, gr2::Grad) = begin
     return all(length(getfield(gr1, k)) ≈ length(getfield(gr2, k)) for k ∈ fieldnames(Grad)) &&
-        all(getfield(gr1, k) ≈ getfield(gr2, k) for k ∈ fieldnames(Grad))
+        all(all(getfield(gr1, k) .≈ getfield(gr2, k)) for k ∈ fieldnames(Grad))
```

Comment:
Not all of the functions need to be in one line, separate to improve readability.